scss/layout/_grid.scss

```diff
@@ -68,6 +69,16 @@
   }
 }
 
+.slab--primary .grid {
+  a {
```

Comment:
Ships crossing in the night! I just commented about some of that, but a separate thought is: if we do need separate styles for the home page slabs, which it seems like we do for the borders and link colors, maybe we should just add `.slab--home` as a modifier class that could be added to `.slab--primary` or `.slab--secondary`.